src/Controls/src/Core/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
+  <assembly fullname="Microsoft.Maui.Controls">
+    <type fullname="Microsoft.Maui.Controls.FeatureFlags">
+      <method signature="System.Boolean get_IsXamlLoadingEnabled()" body="stub" feature="Microsoft.Maui.Controls.FeatureFlags.IsXamlLoadingEnabled" featurevalue="false" />
+    </type>
```

Comment:
I find this a bit easier to read
```suggestion
    <type fullname="Microsoft.Maui.Controls.FeatureFlags" feature="Microsoft.Maui.Controls.FeatureFlags.IsXamlLoadingEnabled" featurevalue="false">
      <method signature="System.Boolean get_IsXamlLoadingEnabled()" body="stub" value="false" />
    </type>
```

Reply:
Makes sense, but I think we should still include the value attribute for clarity.